src/components/sidebar-content-box/SidebarContentBox.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import AppButton from '~/components/app-button/AppButton'
+import AllContentModal from '~/components/all-content-modal/AllContentModal'
+import FileComponent from '~/components/file-component/FileComponent'
+import LinkComponent from '~/components/link-component/LinkComponent'
+
+import { useModalContext } from '~/context/modal-context'
+import { spliceSx } from '~/utils/helper-functions'
+import { SizeEnum, ButtonVariantEnum, Link, File } from '~/types'
+
+import { styles } from '~/components/sidebar-content-box/SidebarContentBox.styles'
+
+interface SidebarContentBoxProps {
+  Icon: React.ComponentType
+  name: string
+  content?: Link[] | File[]
+}
+
+type SidebarImageGridModifications = {
+  compactMode: boolean
+}
+
+const SidebarContentBox: FC<SidebarContentBoxProps> = ({
+  Icon,
+  name,
+  content,
+  children
+}) => {
+  const maxElemToShow = 3
```

Comment:
This is a constant, so you need to save it constant file.

Reply:
> This is a constant, so you need to save it constant file.

fixed